repository/asset_repository.go

```diff
@@ -27,13 +37,67 @@ func NewAssetRepository() AssetRepository {
 	}
 }
 
+func (repo *assetRepo) DeleteAsset(ctx context.Context, Id uuid.UUID) (*domain.Asset, error) {
+	var m domain.Asset
+	var asset domain.Asset
+	err := repo.db.Get(&m, getAssetDelete, Id)
+	if err == sql.ErrNoRows {
+		return nil, customerrors.NoAssetsExist
+	}
+	if m.Status == "retired" {
+		return nil, customerrors.AssetAlreadyDeleted
+	}
+	sample := "retired"
+	tx := repo.db.MustBegin()
+	tx.MustExec(getAssetDeletefun, sample, Id)
+	tx.Commit()
+	err = repo.db.Get(&asset, getAsset, Id)
```

Comment:
why u r getting this again

Reply:
because i have to display updated output in postmen thats why i am getting it again